src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -6,9 +6,16 @@ import {
   PatchTasksQuery,
   PatchTasksVariables
 } from "gql/queries/get-patch-tasks";
+import { Patch } from "gql/queries/patch";
 import { TasksTable } from "pages/patch/patchTabs/tasks/TasksTable";
+import { P2 } from "components/Typography";
 
-export const Tasks: React.FC = () => {
+interface Props {
+  patch: Patch;
```

Comment:
it's good practice to only pass the necessary data as props. Having `patch` as the props gives the impression that the component needs all of the data in `patch` whereas it only needs one string value -- `taskCount`. Passing a single number value `taskCount` as opposed to the entire `patch` object will also cause less re-renders because React can shallowly compare props if all values are primitive -- which objects are not. 

what you could do is use `lodash.get` to access the `taskCount` field on data and then pass that as the prop.

Reply:
that's what I was doing initially, I just wasn't sure if it was correct so I erred on the side of caution and passed the whole object. I'll go back and revert that now.